web/packages/api/src/forKusama.ts

```diff
@@ -137,23 +154,28 @@ export async function getDeliveryFee(
         forwardedXcm = buildTransferPolkadotToKusamaExportXCM(
             sourceAssetHub.registry,
             erc20Location(registry.ethChainId, "0x0000000000000000000000000000000000000000"), // actual token location doesn't matter here, just weighing the message
-            xcmBridgeFee,
-            xcmBridgeFee,
+            xcmBridgeBaseFee,
+            xcmBridgeBaseFee,
             registry.assetHubParaId,
             100000000000n,
             "0x0000000000000000000000000000000000000000000000000000000000000000",
             "0x0000000000000000000000000000000000000000000000000000000000000000"
         )
     }
 
-    // Message on Dest AH
-    //calculateDestinationFee
+    let bytes = forwardedXcm.toHex().length / 2;
```

Comment:
Should we use `forwardedXcm.toU8a().length` instead?

Reply:
Yeah, that is better. https://github.com/Snowfork/snowbridge/pull/1463/commits/df04168327da242314e628ed4dec88e9bbd0bdb6